source/beamline/VESPERS/VESPERSBeamline.cpp

```diff
@@ -1176,6 +1176,60 @@ VESPERSBeamline::~VESPERSBeamline()
 
 }
 
+QString VESPERSBeamline::details() const
+{
+	// Build the notes for the scan.
```

Comment:
Comment could be updated to be specific to the fact that this code now lives in the beamline.